src/website/website.js

```diff
@@ -79,3 +81,39 @@ function readThemeSettings (filepath) {
     throw new Error('Failed to parse JSON theme settings file: ' + filepath)
   }
 }
+
+function outputSEO (output, seoLocation, rootAlbum) {
+  // SEO (sitemap and robots.txt)
+  if (!seoLocation) {
+    return
+  }
+
+  // Guaranteed to end with slash
+  const seoPrefix = seoLocation + (seoLocation.endsWith('/') ? '' : '/')
+
+  // Generate robots.txt
+  const robotsTxt = 'User-Agent: *\nDisallow:\n\nSitemap: ' + seoPrefix + 'sitemap.xml\n'
+  fs.writeFile(path.join(output, 'robots.txt'), robotsTxt, () => {
```

Comment:
Since this method is synchronous, could you switch to `writeFileSync` for the two files? It will ensure any error is thrown/displayed properly in case something happens.

Reply:
I didn't think it existed, but I tried it anyway and it works 👍